charts/pmm/templates/service.yaml

```diff
@@ -1,7 +1,4 @@
 {{- $serviceType := .Values.service.type -}}
-{{- if .Values.ingress.enabled -}}
```

Comment:
I have not test a lot of environments. does nginx ingress require the ClusterIP? on digital ocean it was needed. if that were the case, would it would be useful to have a warning in the NOTES.txt, when not both configured ?
```
{{- if .Values.ingress.enabled }}
...
{{- if not contains "ClusterIP" .Values.service.type  }}
nginx ingress may require a service.type of "ClusterIP"
{{- end }}
...
{{- end }}
```